src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.de.xlf

```diff
@@ -4,387 +4,387 @@
     <body>
       <trans-unit id="ID3000">
         <source>The security token is missing.</source>
-        <target state="new">The security token is missing.</target>
+        <target state="new">Der Security-Token fehlt.</target>
```

Comment:
Can you please replace all the `state` attributes of the translated strings from `new` to `translated`?

Thanks.

Reply:
Done!